src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,54 @@
+package mate.academy.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "cinema_halls")
+public class CinemaHall {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(name = "capacity")
+    private int capacity;
+    @Column(name = "description")
```

Comment:
`JFYI` in such cases it's redundant to add @Column, because under the hood hibernate will make the same naming in tables